WordPress/gutenberg
# Prevent unexpected copying of the post title

@edit: This PR originally attempted to solve the copying problem in the post title and the widget areas at the same time. However, I have decided to resolve the widget issues separately in a new issue #45398. For details, please see the edit history.

## What?
This PR prevents copying of a post titles.

## Why?

When text is selected in the post title, `getSelectedBlockClientIds` is executed.
The post title is a block, but the `clientId` is empty, so the correct block name cannot be obtained at this point:
https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/components/copy-handler/index.js#L34

As a result, a message with an undefined value is displayed.

https://user-images.githubusercontent.com/54422211/170080258-f1df225a-8d52-4a61-b05e-7cbc8c721204.mp4

## How?

Make `getSelectedBlockClientIds` return an empty array if clientId of the selected block is an empty string or undefined value.

## Testing Instructions

- Select one of the letters in the post title.
- Click between the letters to deselect them.
- Press `Ctrl(Cmd)` + `C` or `Ctrl(Cmd)` + `X` .
- **On trunk**: Confirm that the notification message contains "undefined" text.
- **On this branch**: Confirm that no notification message appears and nothing is copied to the clipboard.
